CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [DesignToolbox] Theme selector icon (Orange-OpenSource/ouds-ios#943)
 - [DesignToolbox] [Library] Tuning of themes and theme selector (Orange-OpenSource/ouds-ios#951)
 
+### Fixed
+
+- [Library] Vocalisation of badge with icons (Orange-OpenSource/ouds-ios#956)
```

Comment:
https://github.com/Orange-OpenSource/ouds-ios/issues/956
https://github.com/Orange-OpenSource/ouds-ios/issues/844 are not releated to the PR
and 957 is missing